### ISSUE TITLE
Error saving power data on HPC

### ISSUE THREAD

**Issue by thangdnsf** [georges-da-costa/expetator#2]
Hi,

When I acquis data in HPC I get an error message. I guest the power metrics API request is down or changed.

```
File "/home/thannguyen/.local/lib/python3.7/site-packages/expetator/experiment.py", line 192, in monitor_bench
    self.save_monitors(benchname, beg_time)
  File "/home/thannguyen/.local/lib/python3.7/site-packages/expetator/experiment.py", line 126, in save_monitors
    monitor.save(self, benchname, beg_time)
  File "/home/thannguyen/.local/lib/python3.7/site-packages/expetator/monitors/power.py", line 65, in save
    print(result)
UnboundLocalError: local variable 'result' referenced before assignment
```

**Comment by georges-da-costa** [comment]
Yes, the old system (using power.Power()) does not work any more. The new system on g5k is kwollect.
Expetator 0.1.8 supports it but it can acquire the power measures only using the wattmetre_power_watt attribute which is available on:
- grenoble: troll, yeti, wattmetre1
- lyon: gemini, orion, pyxis, sagittaire, taurus, wattmetre1, wattmetre4
- nancy: gros, gros-wattmetre1, gros-wattmetre2, gros-wattmetre3
Full documentation of kwollect is here: https://www.grid5000.fr/w/Monitoring_Using_Kwollect#Metrics_available_in_Grid.275000

an example of application using the new mechanism:
`#!/usr/bin/python3

import os
import expetator.experiment as experiment

from expetator.benchmarks import sleepbench
from expetator.monitors import kwollect
BENCHMARKS = [sleepbench.SleepBench()]
MONITORS = [kwollect.Power()]

dest_dir = 'kwollect/test'
os.makedirs(os.path.dirname(dest_dir), exist_ok=True)
experiment.run_experiment(dest_dir, BENCHMARKS,
                          leverages=[],
                          monitors=MONITORS,
                          times=1)
`

with the output:
`cat kwollect/test_taurus-4.lyon.grid5000.fr_1623274183_power/taurus-4.lyon.grid5000.fr_sleep-30_1623274188 
[["taurus-4", [1623274188.0, 1623274189.0, 1623274190.0, 1623274191.0, 1623274192.0, 1623274193.0, 1623274194.0, 1623274195.0, 1623274196.0, 1623274197.0, 1623274198.0, 1623274199.0, 1623274200.0, 1623274201.0, 1623274202.0, 1623274203.0, 1623274204.0, 1623274205.0, 1623274206.0, 1623274207.0, 1623274208.0, 1623274209.0, 1623274210.0, 1623274211.0, 1623274212.0, 1623274213.0, 1623274214.0, 1623274215.0, 1623274216.0, 1623274217.0, 1623274218.0], [99.7409090909091, 99.9565217391304, 98.4541666666667, 99.325, 99.7591836734694, 98.4244444444444, 98.072, 98.6291666666667, 98.65, 98.852, 99.5673913043478, 116.831707317073, 100.257777777778, 99.8958333333333, 98.824, 101.1625, 99.0341463414634, 101.036363636364, 98.276, 98.974, 104.442553191489, 110.964583333333, 99.05625, 98.912, 99.8083333333333, 98.84, 117.434782608696, 100.135714285714, 99.7854166666667, 99.354347826087, 99.1863636363637]]]`
